nailgun/entities.py

```diff
@@ -2510,7 +2509,6 @@ def __init__(self, server_config=None, **kwargs):
                 Organization,
                 required=True,
             ),
-            'puppet_module': entity_fields.OneToManyField(PuppetModule),
```

Comment:
I did test this line with `test_positive_create_and_update_with_content_source` that failed before.  It does not fail with `fields and puppet_module anymore`.

Reply:
Awesome, thanks Tomas!